instant-xml-macros/src/se.rs

```diff
@@ -0,0 +1,107 @@
+use std::collections::{BTreeSet, HashMap};
+
+use proc_macro2::TokenStream;
+use quote::quote;
+
+use crate::{get_namespaces, retrieve_field_attribute, FieldAttribute};
+
+pub struct Serializer {
```

Comment:
I'd like to have all the moving code around (and renaming) in a separate PR so that this PR only contains the deserializer changes.

Reply:
I only did this, because Serializer and Deserializer are using the same functions, that were erlier only in Serializer.

Reply:
I'm not objecting to the change itself, just makes it harder for me to review the actually changed code if they're in the same PR.

Reply:
Opened PR: #11